### ISSUE TITLE
jupyter lab / notebook

### ISSUE THREAD

**Issue by profLewis** [jgomezdans/geog_docker#17]
I like lab ... but the extensions arent very good / compatible as yet.

I particular, I like the nbextension exercise2 & want to use this. There is a similar jupyterlab-solutions in lab, but its not compatible with recent labs ... 

So, for now, Id propose we stick (for our course) with notebook. Not that that makes any difference to this repo.

**Comment by jgomezdans** [comment]
I agree that extensions are very hit & miss. I don't mind whether we use notebook or lab, but it does result in a small easy tweak. Note that by keeping the notebook we miss out on the fancy debugger... choices! choices! ;)
